lib/public/IMemcache.php

```diff
@@ -67,10 +67,24 @@ public function cas($key, $old, $new);
 	/**
 	 * Compare and delete
 	 *
+	 *  Delete $key if the stored value is equal to $old
+	 *
 	 * @param string $key
 	 * @param mixed $old
 	 * @return bool
 	 * @since 8.1.0
 	 */
 	public function cad($key, $old);
+
+	/**
+	 * Negative compare and delete
+	 *
+	 * Delete $key if the stored value is not equal to $old
+	 *
+	 * @param string $key
+	 * @param mixed $old
+	 * @return bool
+	 * @since 30.0.0
+	 */
+	public function ncad(string $key, $old): bool;
```

Comment:
```suggestion
	public function ncad(string $key, mixed $old): bool;
```
In these cases where we really accept anything we should strongtype it as well, `mixed` is available in `PHP>=8.0`